### ISSUE TITLE
NO output from powershell when an argument value changes

### ISSUE THREAD

**Issue by gdharanidhar** [rannn505/node-powershell#31]
Hello

I am trying to execute a simple get-aduser command and it works fine for most of the users.
But for some users the output is null.

When I log the command being invoked and execute from a standalone powershell it works just fine.

Thanks!

**Comment by gdharanidhar** [comment]
Sample Output below:

As you can see for the userId 'goxxxid' I actually get results, but for the first ID 'prxxxxrp' it sends out null.
When I copy pase the commands (asterisked below) below in a powershell window, both of them return results. 

```
[nodemon] restarting due to changes...
[nodemon] starting `node app.js`
NPS>  Process 2300 started

Running
NPS>  Command invoke started
******c:\temp\test.ps1 prxxxxrp memberof******
NPS>  Command invoke finished


[]
[nodemon] restarting due to changes...
[nodemon] starting `node app.js`
NPS>  Process 2856 started

Running
NPS>  Command invoke started
 ******c:\temp\test.ps1 gorxxxid memberof******
NPS>  Command invoke finished


{
    "memberof":  [
```

**Comment by BenjaminMichael** [comment]
This was fixed in Issue #20
